paddle/fluid/operators/reader/create_double_buffer_reader_op.cc

```diff
@@ -42,7 +62,10 @@ class DoubleBufferReader : public framework::DecoratedReader {
  private:
   void PrefetchThreadFunc();
 
-  framework::Channel<std::vector<framework::LoDTensor>>* buffer_;
+  framework::Channel<Item>* buffer_;
+  platform::Place place_;
+  std::vector<std::unique_ptr<platform::DeviceContext>> ctxs_;
```

Comment:
The `DoubleBufferReader` only takes one `Place` and all ctx would be the same. So why do we need a vector here?